test/Microsoft.AspNetCore.Razor.Test/CodeGenerators/CSharpCodeGeneratorTest.cs

```diff
@@ -55,13 +65,123 @@ public void ChunkTreeWithUsings()
             if (!string.Equals(expectedOutput, result.Code, StringComparison.Ordinal))
             {
                 BaselineWriter.WriteBaseline(
-                    @"test\Microsoft.AspNetCore.Razor.Test\TestFiles\CodeGenerator\Output\CSharpCodeGenerator.cs",
+                    Path.Combine(_testOutputDirectory, "Output", "CSharpCodeGenerator.cs"),
                     result.Code);
             }
 #else
             Assert.Equal(expectedOutput, result.Code);
 #endif
         }
+
+        public static TheoryData<CSharpCodeGenerator, CodeGeneratorResult> ModifyOutputData
+        {
+            get
+            {
+                var addGenerator = new DecorateChunksAddTestCodeGenerator(CreateContext());
+                var clearGenerator = new DecorateChunksClearTestCodeGenerator(CreateContext());
+                var defaultGenerator = new CSharpCodeGenerator(CreateContext());
+                var commentGenerator = new BuildAfterExecuteContentTestCodeGenerator(CreateContext());
+
+                var addResult = CreateCodeGeneratorResult("AddDecorateChunkTest.cs");
+                var clearResult = CreateCodeGeneratorResult("ClearDecorateChunkTest.cs");
+                var defaultResult = CreateCodeGeneratorResult("DefaultDecorateChunkTest.cs");
+                var commentResult = CreateCodeGeneratorResult("BuildAfterExecuteContentTest.cs");
+
+                var theoryData = new TheoryData<CSharpCodeGenerator, CodeGeneratorResult>
+                {
+                    {addGenerator, addResult },
+                    {clearGenerator, clearResult },
+                    {defaultGenerator, defaultResult },
+                    {commentGenerator, commentResult }
+                };
+                
+                return theoryData;
+            }
+        }
+        
+        [Theory]
+        [MemberData(nameof(ModifyOutputData))]
+        public void BuildAfterExecuteContent_DecorateChunks_ModifiesOutput(
```

Comment:
Suggest `BuildAfterExecuteContent_DecorateChunks_ModifyOutput()` since this name mentions two methods.